docs/environment-variables.md

```diff
@@ -63,25 +63,38 @@ The OAuth token secret, generated by Launchpad.
 ## GitHub settings
 
 ### GITHUB_API_ENDPOINT
-- **Example**: `http://localhost:4000/github`, `https://api.github.com`
+- **Example**: `http://localhost:4000/github`
+- **Example**: `https://api.github.com`
 
```

Comment:
No big change here, just re-formatting examples over two lines to match changes later in the file.